components/layout/index.js

```diff
@@ -0,0 +1,30 @@
+import Head from 'next/head';
+import PropTypes from 'prop-types';
+
+const Layout = (props) => {
+  const { children, title, description } = props;
+  return (
+    <div>
+      <Head>
+        <title>{title}</title>
+        <meta name="description" content={description} />
+      </Head>
+      <div className="container">
+        <div>{children}</div>
+      </div>
```

Comment:
If we have a layout, can't we have the `Footer` here? We can give the `.container` a min-height of `95vh`, so that footer is always at the bottom. Coordinate with @manjots1607

Reply:
Better I'll talk to him.